.travis.yml

```diff
@@ -39,12 +39,17 @@ before_install:
   - export DOTNET_ROOT=/home/travis/.dotnet
   - export PATH=/home/travis/.dotnet:/home/travis/.dotnet/tools:$PATH
   - dotnet --info
+  - wget -O /tmp/black https://github.com/python/black/releases/download/19.3b0/black
+  - chmod +x /tmp/black
+  - /tmp/black --version
 
 install:
   # Install the code requirements
   - make init
 
 script:
+  # Verify code is black compliant
+  - /tmp/black --check setup.py tests samcli
```

Comment:
`make black` instead?

Reply:
`make black` will format the files are write it back to the filesystem. Instead this commands `black --check` makes sure that the files are formatted and if not the build will fail. This ensures our codebase is always formatted. I can create a `make black-check`? command or something. I thought about putting it in `make pr` but didn't want to go through the hassle of pip installing on appveyor too (you can only [pip] install black in 3.6 or above).